Mlem/Views/Shared/Comments/Comment Item.swift

```diff
@@ -184,6 +187,21 @@ struct CommentItem: View {
                     Spacer()
                         .frame(height: AppConstants.postAndCommentSpacing)
                 }
+                
+                if collapseComments,
+                   !hierarchicalComment.isCollapsed,
+                   hierarchicalComment.depth == 0,
+                   hierarchicalComment.commentView.counts.childCount > 0,
+                   !isCommentReplyHidden {
+                    HStack {
+                        Divider()
+                        CollapsedCommentReplies(numberOfReplies: .constant(hierarchicalComment.commentView.counts.childCount))
```

Comment:
This Divider() looks better outside the HStack, I think

```suggestion
                   !isCommentReplyHidden {
                    Divider()
                    HStack {                        CollapsedCommentReplies(numberOfReplies: .constant(hierarchicalComment.commentView.counts.childCount))
```